README.md

```diff
@@ -282,6 +284,38 @@ use:
 default_mimetype=text/plain
 ```
 
+If the default MIME type isn't explicitly configured in YAML configuration,
+then the `freezeyt` uses value `application/octet-stream`.
+
+#### Recognizing file types from extensions
+
+There is possibility to modify the way how to determine file type
+from file extension.
+You can setup your own `get_mimetype` function.
+
+Freezeyt will register your own function, if you specify it in configuration
+YAML file as:
+
+```yaml
+get_mimetype=module:your_function
+```
+
+If the `get_mimetype` is not defined in configuration file,
+then the `freezeyt` uses the python function `mimetypes.guess_type`
+
+`get_mimetype` can be defined as:
+* strings in the form `"module:function"`, which name the function to call,
+* Python functions (if configuring `freezeyt` from Python, e.g. as a `dict`,
+  rather than YAML).
+
+The `get_mimetype`:
+*gets one argument the `filepath` as `string`
+
+*returns file type as `string` (e.g. `"text/plain"`).
```

Comment:
We have conflict with what should return our `get_mimetype` and what is returning func guess_mimetype.